vms/components/multisig/camino_multisig.go

```diff
@@ -41,3 +41,11 @@ func (ma *Alias) VerifyState() error {
 func ComputeAliasID(txID ids.ID) ids.ShortID {
 	return hashing.ComputeHash160Array(txID[:])
 }
+
+// AliasDef is the definition of a Multisig alias used for storage
+type AliasDef struct {
```

Comment:
Instead of creating another type, we should modify Alias type accordingly. Alias type will be used both in state and in txs that will create/update msig aliases - and we need pub keys for that.